src/rmallexn.rs

```diff
@@ -0,0 +1,115 @@
+use std::fs::{self, ReadDir};
+use std::io::Error;
+use std::path::Path;
+use std::{path::PathBuf};
+
+pub fn rmxn(command: String) {
+    
+    let command = command;
+
+    if command.starts_with("rmallexn"){
+
+        //splitting on whitespace
+        let chunks: Vec<_> = command.split_whitespace().collect();
+
+        //checking if command is an exact match else suggest using rmallexn command
+        if chunks[0] == String::from("rmallexn"){
+
+            //println!("{:?}", chunks);
```

Comment:
remove all the commented out code in this file

Reply:
commented code removed